recap/converters/json_schema.py

```diff
@@ -262,8 +267,8 @@ def _convert_field(
                 return self._convert_field(field.resolve())
             case ProxyType(alias=str(alias)):
                 return type_dict | {"$ref": f"#/$defs/{alias}"}
-            case _:
-                raise ValueError(f"Unsupported RecapType: {field.type_}")
+            case UnknownType():
```

Comment:
I feel like we should still raise value error here.. Same as I commented in Avro converter.